packages/gui/src/components/vcs/VCCard.tsx

```diff
@@ -84,10 +84,14 @@ export default function VCCard(props: { vcRecord: any; isDetail?: boolean; proof
   }
 
   function renderProperties() {
-    const didString = vcRecord?.vc?.proofProvider
-      ? didToDIDId(vcRecord?.vc?.proofProvider)
-      : vcRecord.credentialSubject?.id || '/';
-
+    let didString: string = vcRecord.credentialSubject?.id || '/';
+    try {
+      if (vcRecord?.vc?.proofProvider) {
+        didString = didToDIDId(vcRecord?.vc?.proofProvider);
```

Comment:
nit: assign `vcRecord?.vc?.proofProvider` to a local variable instead of copying & pasting

Reply:
Fixed.